docs/using_eessi/eessi_demos.md

```diff
@@ -32,30 +32,43 @@ Each of these contains a ``run.sh`` script that can be used to start a small
1s - loss: 0.0715 - accuracy:  613/1875 [========>.....................] - ETA: 1s - loss: 0.0714 - accuracy:  649/1875 [=========>....................] - ETA: 1s - loss: 0.0719 - accuracy:  685/1875 [=========>....................] - ETA: 1s - loss: 0.0730 - accuracy:  721/1875 [==========>...................] - ETA: 1s - loss: 0.0744 - accuracy:  757/1875 [===========>..................] - ETA: 1s - loss: 0.0735 - accuracy:  793/1875 [===========>..................] - ETA: 1s - loss: 0.0731 - accuracy:  829/1875 [============>.................] - ETA: 1s - loss: 0.0726 - accuracy:  865/1875 [============>.................] - ETA: 1s - loss: 0.0725 - accuracy:  901/1875 [=============>................] - ETA: 1s - loss: 0.0731 - accuracy:  937/1875 [=============>................] - ETA: 1s - loss: 0.0730 - accuracy:  973/1875 [==============>...............] - ETA: 1s - loss: 0.0731 - accuracy: 1009/1875 [===============>..............] - ETA: 1s - loss: 0.0733 - accuracy: 1045/1875 [===============>..............] - ETA: 1s - loss: 0.0731 - accuracy: 1081/1875 [================>.............] - ETA: 1s - loss: 0.0730 - accuracy: 1117/1875 [================>.............] - ETA: 1s - loss: 0.0724 - accuracy: 1153/1875 [=================>............] - ETA: 1s - loss: 0.0716 - accuracy: 1189/1875 [==================>...........] - ETA: 0s - loss: 0.0713 - accuracy: 1225/1875 [==================>...........] - ETA: 0s - loss: 0.0711 - accuracy: 1261/1875 [===================>..........] - ETA: 0s - loss: 0.0714 - accuracy: 1297/1875 [===================>..........] - ETA: 0s - loss: 0.0717 - accuracy: 1333/1875 [====================>.........] - ETA: 0s - loss: 0.0715 - accuracy: 1369/1875 [====================>.........] - ETA: 0s - loss: 0.0719 - accuracy: 1405/1875 [=====================>........] - ETA: 0s - loss: 0.0718 - accuracy: 1441/1875 [======================>.......] - ETA: 0s - loss: 0.0724 - accuracy: 1477/1875 [======================>.......] - ETA: 0s - loss: 0.0726 - accuracy: 1513/1875 [=======================>......] - ETA: 0s - loss: 0.0729 - accuracy: 1549/1875 [=======================>......] - ETA: 0s - loss: 0.0727 - accuracy: 1585/1875 [========================>.....] - ETA: 0s - loss: 0.0729 - accuracy: 1621/1875 [========================>.....] - ETA: 0s - loss: 0.0732 - accuracy: 1657/1875 [=========================>....] - ETA: 0s - loss: 0.0735 - accuracy: 1693/1875 [==========================>...] - ETA: 0s - loss: 0.0739 - accuracy: 1729/1875 [==========================>...] - ETA: 0s - loss: 0.0740 - accuracy: 1765/1875 [===========================>..] - ETA: 0s - loss: 0.0739 - accuracy: 1801/1875 [===========================>..] - ETA: 0s - loss: 0.0736 - accuracy: 1837/1875 [============================>.] - ETA: 0s - loss: 0.0736 - accuracy: 1873/1875 [============================>.] - ETA: 0s - loss: 0.0732 - accuracy: 1875/1875 [==============================] - 3s 1ms/step - loss: 0.0732 - accuracy: 0.9772
```

Comment:
This is showing all "updates" per line, I think it's fine to just copy-paste the output once the command is done running?

Reply:
This is what it looks like on the web page; It is not printing it line by line in the markdown

![Screenshot (122)](https://github.com/EESSI/docs/assets/49882639/cec1c246-24e5-47f9-9fb4-e9f520a6a374)

Reply:
OK, sure, but those hugely long lines are not very useful (and they're cut off too, see the `accurac ` part for 1st epoch).

I would use this instead (also removed those first 2-3 lines):

```
Epoch 1/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.2894 - accuracy: 0.9158
Epoch 2/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.1418 - accuracy: 0.9585
Epoch 3/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.1073 - accuracy: 0.9668
Epoch 4/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.0862 - accuracy: 0.9740
Epoch 5/5
1875/1875 [==============================] - 3s 2ms/step - loss: 0.0738 - accuracy: 0.9763
313/313 - 0s - loss: 0.0723 - accuracy: 0.9775 - 320ms/epoch - 1ms/step

real    0m21.395s
user    0m32.558s
sys     0m3.303s
```